WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java

```diff
@@ -1369,6 +1370,11 @@ public static void viewAppSettingsForResult(Activity activity) {
         activity.startActivityForResult(intent, RequestCodes.APP_SETTINGS);
     }
 
+    public static void viewDesignSystemForResult(Activity activity) {
+        Intent intent = new Intent(activity, DesignSystemActivityCompose.class);
+        activity.startActivityForResult(intent, RequestCodes.DESIGN_SYSTEM_SETTINGS);
```

Comment:
What is `startActivityForResult` for here?  Is this going to do anything with the result?  If not, just `startActivity` could be enough.

Reply:
Nice catch, updated 👍